src/Runner.Listener/BrokerMessageListener.cs

```diff
@@ -36,13 +44,129 @@ public override void Initialize(IHostContext hostContext)
 
         public async Task<Boolean> CreateSessionAsync(CancellationToken token)
         {
-            await RefreshBrokerConnection();
-            return await Task.FromResult(true);
+            Trace.Entering();
```

Comment:
this is all copied from MessageListener. Eventually, one of these will be deprecated, but keeping them in sync for now